project/hychiu02/README.md

```diff
@@ -0,0 +1,96 @@
+# AutoEncoder for arrhythmia detection
+
+## Basic information
+
+[GitHub](https://github.com/hychiu02/AutoEncoder-for-arrhythmia-detection)
+
+Arrhythmia dectection using ECG signal
+
+## Problem to solve
+Heart diseases are one of the significant reasons for death all over the planet. To perform arrhythmia detection, Electrocardiograms(ECGs) is used to monitor hte functioning of heart by capturing electrical activity. Because of needs of real-time inference, the model sizeand  parameters should be small, autoencoder seems to be a good choice. I want to develop a method to build, train and inference a MLP model with C++. 
+
+## Prospective Users
+People who want to build Multilayer Perceptron model for particular case.

```

Comment:
You can be more specific about the particular cases.  One of the use is the testing and software engineering I mentioned above.

Reply:
Perhaps I misunderstand the meaning of prospective users, I think people who want to do abnormal sound detection, cardiac diseases classification or other regression or classification problem can take advantage of my implementation.

Reply:
You did not misunderstand.  Yes, and please put that in your proposal.  That will make the development process clearer.  And you know what you change when you need to change.

Reply:
Ok, I add into my proposal.Thank you!